portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,71 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto+Mono:300,400");
+@import url("https://fonts.googleapis.com/css?family=Josefin+Sans:300,400,700");
+html {
+  padding: 0px;
+  margin: 0px;
```

Comment:
nit: alphabetize the properties. Makes it more readable when you have a larger number of properties.

Reply:
I alphabetized the css properties. Please, let me know if it looks fine now :)